docs/docusaurus/versioned_sidebars/version-1.7.0-sidebars.json

```diff
@@ -117,7 +117,7 @@
         "type": "subcategory",
         "label": "NMS",
         "ids": [
-          "version-1.7.0-nms/deploy_setup"
```

Comment:
Isn't this one already correct, it should be setup right?

Reply:
Oh yes, thanks for spotting it! I somehow didn't commit that file